nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -668,7 +668,7 @@ def pylint(context):
 @task(aliases=("a",))
 def autoformat(context):
```

Comment:
Do we need to have 2 ways to do the same thing?

- `invoke autoformat` 
- `invoke ruff --action format`

Also, core and apps have diverged on the `invoke ruff` commands, can we standardize on one?

Reply:
I will always use invoke autoformat. I think the other invoke ruff is for testing. But, I wouldn't mind matching Core.

Reply:
As I can see `autoformat` is an alias to the `ruff` task, I don't see a problem since there's no code duplication.

Reply:
> Also, core and apps have diverged on the invoke ruff commands, can we standardize on one?

Main difference seems to be in the `output_format` default?

Ah and the `action` which doesn't exist for core - I'd rather be able to decide if I want linting or just formatting. But the default could be lint+format so the action is the same in core and apps.

Core also has `target`, but I'm not sure it's needed in apps where we check the whole project folder.

Reply:
> I will always use invoke autoformat. I think the other invoke ruff is for testing. But, I wouldn't mind matching Core.

Wasn't the `invoke autoformat` command introduced relatively recently with ruff? Before that it was `invoke pydocstyle` for apps?

Reply:
@cmsirbu As discussed, we'll also change the default action for `invoke ruff` (with no supplied `--action`) to `lint` and `format` to match core.